configs/_base_/models/tsn_r18.py

```diff
@@ -0,0 +1,19 @@
+# model settings
+model = dict(
+    type='Recognizer2D',
+    backbone=dict(
+        type='ResNet',
+        pretrained='torchvision://resnet18',
+        depth=18,
```

Comment:
`r18` model may not be included in `_base_` since there is `r50` one.

Reply:
okk